### PR TITLE
Fix docs according to 2 modified endpoints (that now return coverage completeness as well)

### LINKED ISSUE #151: Fix docs according to modified endpoints
Fix documentation according to changes introduces in #148 and #150

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Fix #151 

**How to prepare for test**:
- [ ] `ssh` to ...
- [ ] Install on stage:
`bash servers/resources/SERVER.scilifelab.se/update-[THIS_TOOL]-stage.sh [THIS-BRANCH-NAME]`

### How to test:
- Tests executed locally to create the howto content

### Review:
- [x] Code approved by HS
- [x] Tests executed by CR

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -21,13 +21,13 @@ docker run --entrypoint d4tools --rm  clinicalgenomics/chanjo2:latest
 ### Calculating coverage on specific genomic intervals of a d4 file using d4tools
 
 ``` shell
-docker run --entrypoint d4tools --rm  -v <path-to-local-d4-files-folder>:/home/worker/infiles clinicalgenomics/chanjo2:latest view /home/worker/infiles/<d4file.d4> 1:1234560-1234580 X:1234560-1234580
+docker run --entrypoint d4tools --platform linux/x86_64 --rm  -v <path-to-local-d4-files-folder>:/home/worker/infiles clinicalgenomics/chanjo2:latest view /home/worker/infiles/<d4file.d4> 1:1234560-1234580 X:1234560-1234580
```
Comment (northwestwitch) [inline docs/index.md:24]: This is only for those starting the container from a computer with the M1 and M2 chip (like me)

File: docs/usage/sample-coverage.md [hunk 2]
```diff
@@ -3,141 +3,169 @@
 Chanjo2 can be used to quickly access average coverage depth statistics for an interval from a list of genomic intervals on a custom d4 coverage file.
 Additionally, samples stored in the database contain the location of d4 files on disk or on a remote server. For this reason, it is possible to retrieve coverage statistics relative to one or more samples by specifying their name (or the case name).
 
+### Coverage completeness
```
Comment (northwestwitch) [inline docs/usage/sample-coverage.md:6]: I moved this up from line 136 in the same document

File: docs/usage/sample-coverage.md [hunk 3]
```diff
@@ -3,141 +3,169 @@
...
-    ],
+    "mean_coverage": {
+      "D4File": 141.35853114545165
+    },
     "start": 26721661
   }
 ]
 ```
 
-Please note that <strong>direct coverage queries are not yet supporting the computation of coverage completeness</strong> over a custom list of coverage thresholds.
```
Comment (northwestwitch) [inline docs/usage/sample-coverage.md:133]: This is a lie now! :)

**Review by henrikstranneheim — APPROVED** [review]
Great 👍

**Pull request merged**